content/app/development/logic/expressions/_index.nb.md

```diff
@@ -119,19 +119,54 @@ Og for en person som er 15 år (eller yngre, som f.eks. en 4-åring), returneres
-| [Repeterende grupper](../../ux/fields/grouping/repeating) | `edit.saveAndNextButton` * | [Boolsk](#boolske-verdier) | ✅        | ❌       |
-
-\* = Disse egenskapene kan foreløpig bare styres dynamisk for alle [repeterende grupper](../../ux/fields/grouping/repeating)
-på en gang, ikke for hver enkelt rad.
+| Komponenter                                               | Egenskap                                        | Forventet verdi            | Frontend | Backend |
+|-----------------------------------------------------------|-------------------------------------------------|----------------------------|----------|---------|
+| [Sider/layouts](#viseskjule-hele-sider)                   | `hidden`                                        | [Boolsk](#boolske-verdier) | ✅        | ✅       |
+| Alle                                                      | `hidden`                                        | [Boolsk](#boolske-verdier) | ✅        | ✅       |
+| Skjemakomponenter                                         | `required`                                      | [Boolsk](#boolske-verdier) | ✅        | ✅       |
+| Skjemakomponenter                                         | `readOnly`                                      | [Boolsk](#boolske-verdier) | ✅        | ❌       |
+| [Repeterende grupper](../../ux/fields/grouping/repeating) | `edit.addButton`                                | [Boolsk](#boolske-verdier) | ✅        | ❌       |
+| [Repeterende grupper](../../ux/fields/grouping/repeating) | `edit.saveButton` *                             | [Boolsk](#boolske-verdier) | ✅        | ❌       |
+| [Repeterende grupper](../../ux/fields/grouping/repeating) | `edit.deleteButton` *                           | [Boolsk](#boolske-verdier) | ✅        | ❌       |
+| [Repeterende grupper](../../ux/fields/grouping/repeating) | `edit.saveAndNextButton` *                      | [Boolsk](#boolske-verdier) | ✅        | ❌       |
+| Alle                                                      | `textResourceBindings.[textResourceBinding]` *  | [Streng](#strenger)        | ✅        | ❌       |
+
+\* = Hvilke verdier man kan overstyre med textResourceBindings varierer fra komponent til komponent, men vil fungere på
+alle steder der det brukes. TextResourceBindigs for repeterende grupper finner du [her](../../ux/fields/grouping/setup#textresourcebindings)
+
+{{% expandlarge id="rep-group-edit-button-text" header="Eksempel: Styre redigeringsknapp-tekst i repeterende gruppe" %}}
+
+Her endrer vi teksten til redigeringsknappen i en repeterende gruppe basert på om `IsPrefill` er satt til `true` i
+en gitt adresse i datamodellen. Dersom `IsPrefill` er `true` for en adresse, vil raden som viser frem den adressen ha en
+redigerings-knapp med teksten `"View"`. Hvis `IsPrefill` er `false` blir teksten på knappen til den spesifikke raden
+`"Edit"`. Det er verdt å merke seg at dersom `IsPrefill` ikke er satt til hverken `true` eller `false` så vil uttrykket
+tolke verdien som `false`.
```

Comment:
Uttrykket vil ikke tolke verdien som `false`, men et oppslag i datamodellen som gir resultatet `null` (ikke funnet) vil konverteres til `false` når det blir brukt i en `if` (siden den forventer datatypen `boolean` i første parameter). Det finnes utfyllende dokumentasjon om det under datatyper lengre nede på siden, og for `if`-funksjonen, så du kan gjerne lenke dit.